cpp/src/utility/quadtree_thrust.cuh

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <thrust/functional.h>
+#include <thrust/pair.h>
+#include <thrust/tuple.h>
+
+#include "utility/bbox_thrust.cuh"
+#include "utility/z_order.cuh"
+
+namespace {
+
+template <typename T>
+struct xytoz {
```

Comment:
For consistency with the other conversion functors.
```suggestion
struct xy2z {
```

Reply:
This type isn't used anymore, but I've kept it here in case the other related in-flight PRs use it.